pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/showcases/multiplayer/MultiPlayerViewModel.kt

```diff
@@ -21,8 +21,14 @@ class MultiPlayerViewModel(application: Application) : AndroidViewModel(applicat
     private val player2 = PlayerModule.provideDefaultPlayer(application)
 
     init {
-        player1.setMediaItem(MediaItem.Builder().setMediaId("urn:rts:video:6820736").build())
-        player2.setMediaItem(MediaItem.fromUri("https://storage.googleapis.com/wvmedia/clear/hevc/tears/tears.mpd"))
+        /*
+         * On some device playing drm content on multiple player may not work.
+         * One of the player will receive a PlaybackException with ERROR_CODE_DECODER_INIT_FAILED.
+         * It may happen on low hand device like :
+         *  - Samsung Galaxy A13
+         */
```

Comment:
```suggestion
        /*
         * On some devices playing DRM content on multiple players may not work.
         * One of the players will receive a PlaybackException with ERROR_CODE_DECODER_INIT_FAILED.
         * It may happen on low-end devices like Samsung Galaxy A13, for example.
         */
```